cloner.py

```diff
@@ -0,0 +1,25 @@
+# clone multiple course repos
+
+from git import Repo
+
+user = "t1s1"
+token = "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"
```

Comment:
I would assume this token is now compromised

Reply:
it's expired but good point